busy-app/busybar-firmware
# Input refactoring

# What's new

- Simplify folder structure
- Improve f20 input performance
- Fix graphical glitches on OLED display
- Update `keypad_test` app

# How to test

1. Install hardware serial flow control mod
2. Flash both f20 and f64 firmware
3. Use the encoder to select the `Keypad Test` option
4. Press the buttons, rotate the encoder, flip the mode switch, the screen should reflect that
